AppController/lib/nginx.rb

```diff
@@ -356,69 +351,68 @@ def self.create_datastore_server_config(all_private_ips, proxy_port)
     
 server {
     listen #{DatastoreServer::LISTEN_PORT_NO_SSL};
-    root /root/appscale/AppDB/;
+    root   /root/appscale/AppDB/;
```

Comment:
@obino these paths are not defined as constants in our ruby pieces. I had a different branch for creating a config file to include in existing files and replace the hardcoded paths.

It's not ready though.